.CI/create_feedstocks.py

```diff
@@ -254,10 +252,24 @@ def write_token(name, token):
                 team.url + "/memberships/" + new_member
             )
 
+    # Update status based on remote
+    subprocess.check_call(['git', 'checkout', os.environ.get('TRAVIS_BRANCH')])
+    subprocess.check_call(['git', 'pull', '--rebase', '--autostash'])
+
+    # Generate a fresh listing of recipes removed.
+    # This gets pretty ugly as we parse `git status --porcelain`.
+    #
+    # * Each line we get back is a change to a file in the recipe directory.
+    # * We narrow the list down to recipes that are staged for deletion (ignores examples).
+    # * Then we clean up the list so that it only has the recipe names.
+    removed_recipes = subprocess.check_output(['git', 'status', '--porcelain', recipe_directory_name])
+    removed_recipes = removed_recipes.splitlines()
+    removed_recipes = filter(lambda _: _.startswith("D "), removed_recipes)
+    removed_recipes = map(lambda _ : _.replace("D", "", 1).lstrip(), removed_recipes)
```

Comment:
Fortunately caught this before the CI did, but not soon enough to avoid it in this PR. Need to convert this to a `list` on Python 3. :facepalm:

xref: https://github.com/conda-forge/staged-recipes/pull/1470